### PR TITLE
MC:11891 Delete router docs + error correction in VpnTunnel docs

### PR DESCRIPTION
**Pull request by Shabirmean**
#### Changes made
- Added docs for Delete ROuter
- Fix errors in the VpnTunnel docs

#### Related PRs
- PR [#254](https://github.com/cloudops/cloudmc-gcp-plugin/pull/254) of [cloudmc-gcp-plugin](https://github.com/cloudops/cloudmc-gcp-plugin)

### CODE REVIEW

**Review by y0geshdev — APPROVED** [review]
Easy peasy. 👍

File: source/includes/gcp/_vpn_tunnels.md [hunk 1]
```diff
@@ -185,9 +185,9 @@ Optional | &nbsp;
 `description`<br/>*string* | An optional description.
 `localTrafficSelector`<br/>*string* | Local traffic selector to use when establishing the VPN tunnel with the peer VPN gateway. The value should be a CIDR formatted string.
```
Comment (Gerudhoh) [inline source/includes/gcp/_vpn_tunnels.md:186]: this line still says gateway ;)

File: source/includes/gcp/_routers.md [hunk 2]
```diff
@@ -243,5 +243,23 @@ Add one or more BGP peers associated to existing router interface(s)
 | `bgpPeers[*].name`<br/>_string_          | The name of the BGP peer association.                                                        |
 | `bgpPeers[*].interfaceName`<br/>_string_ | The router interface through which this BGP peering will be realized.                        |
 | `bgpPeers[*].ipAddress`<br/>_string_     | The router's IP address on the above interface for this BGP peering. Only IPv4 is supported. |
-| `bgpPeers[*].peerIpAddress`<br/>_string_ | The peer IP address this BGP peering tries to connect to. Only IPv4 is supported.   |
+| `bgpPeers[*].peerIpAddress`<br/>_string_ | The peer IP address this BGP peering tries to connect to. Only IPv4 is supported.            |
```
Comment (Gerudhoh) [inline source/includes/gcp/_routers.md]: Could we perhaps change this to "The peer IP address that the BGP peering will try to connect to. Only IPv4 is supported."

File: source/includes/gcp/_routers.md [hunk 3]
```diff
@@ -243,5 +243,23 @@ Add one or more BGP peers associated to existing router interface(s)
 | `bgpPeers[*].name`<br/>_string_          | The name of the BGP peer association.                                                        |
 | `bgpPeers[*].interfaceName`<br/>_string_ | The router interface through which this BGP peering will be realized.                        |
```
Comment (Gerudhoh) [inline source/includes/gcp/_routers.md:244]: Would you mind clarifying something for me, Shabir? Is "peering" the preferred industry word? 
Right now it doesn't quite sit well with me, but if it's the best way then it's the best way!

File: source/includes/gcp/_routers.md [hunk 4]
```diff
@@ -160,7 +160,7 @@ Create a new router
 | --------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
 | `name`<br/>_string_         | The display name of the router.                                                                                                 |
 | `shortRegion`<br/>_string_  | A short version of the region name.                                                                                             |
-| `shortNetwork`<br/>_string_ | A short version of the network name this router is attached to.                                                        |
+| `shortNetwork`<br/>_string_ | A short version of the network name this router is attached to.                                                                 |
```
Comment (Gerudhoh) [inline source/includes/gcp/_routers.md]: Would you mind changing this to "A shortened version of the network name that the router is attached to"?

File: source/includes/gcp/_routers.md [hunk 5]
```diff
@@ -160,7 +160,7 @@ Create a new router
...
 | `bgp.asn`<br/>_string_      | The BGP Autonomous System Number (ASN) Number that is to be assigned to this router. _Must be a valid ASN between 64512–65534_. |
```
Comment (Gerudhoh) [inline source/includes/gcp/_routers.md]: "Number (ASN) Number". Is it alright if we ditch the second time we use the word "Number"?

File: source/includes/gcp/_vpn_tunnels.md (hunk 1 above)
Comment (Shabirmean) [reply source/includes/gcp/_vpn_tunnels.md:186]: If I am not mistaken the use of vpnGateway is correct here. The tunnel is connecting to the peer-gateway.

File: source/includes/gcp/_routers.md (hunk 2 above)
Comment (Shabirmean) [reply source/includes/gcp/_routers.md]: Changed in 3573f35ba6eea283f9bc657b837f7e2601e9be49

File: source/includes/gcp/_routers.md (hunk 4 above)
Comment (Shabirmean) [reply source/includes/gcp/_routers.md]: Changed in 3573f35ba6eea283f9bc657b837f7e2601e9be49

File: source/includes/gcp/_routers.md (hunk 3 above)
Comment (Shabirmean) [reply source/includes/gcp/_routers.md:244]: @Gerudhoh That's right. To create a direct connect between an on=prem network and a cloud network you have to do a "Peering" between the edge devices. And that's a specific term.

File: source/includes/gcp/_routers.md (hunk 5 above)
Comment (Shabirmean) [reply source/includes/gcp/_routers.md]: Ofcourse xD. My bad;
Fixed in 7e73a1e91a08ace8091352d9798a376915815c5a

File: source/includes/gcp/_vpn_tunnels.md (hunk 1 above)
Comment (Gerudhoh) [reply source/includes/gcp/_vpn_tunnels.md:186]: okie dokie ! :D you know more than me!

File: source/includes/gcp/_routers.md (hunk 3 above)
Comment (Gerudhoh) [reply source/includes/gcp/_routers.md:244]: I now know my least favourite term 😛

Comment (Shabirmean) [reply source/includes/gcp/_routers.md:244]: Hahahahah Lol!!!

**Review by Gerudhoh — APPROVED** [review]
LGTM!

**Pull request merged**